src/object/player.cpp

```diff
@@ -2324,8 +2325,13 @@ Player::collision_solid(const CollisionHit& hit)
 
   // crushed?
   if (hit.crush) {
-    if (hit.left || hit.right) {
-      kill(true);
+    if ((hit.left || hit.right)) {
+      if (m_ignore_sideways_crush) {
+        m_ignore_sideways_crush = false;
+      }
+      else {
+        kill(true);
+      }
```

Comment:
If tux isn't killed immediately when crushed vertically, then why can't it also happen here? This way you remove the ignore_sidways_crush thing. But I might be missing something